### DISCUSSION TITLE
A question about DTM embeddings

### DISCUSSION THREAD

**Discussion by lihaoming120309** [MaartenGr/BERTopic#2233] [Q&A] [answered]
Dear authors:
Could i get embeddings of each topic at each timestamp?

**Comment by MaartenGr** [comment, marked as answer]
@lihaoming120309 There are no embeddings of each topic at each timestamp. They are currently created through the c-TF-IDF matrices. If you want their intermediate representation, you would have to adjust the code manually.

**Reply by lihaoming120309** [reply]
Thanks again for your answer and your attribution on bertopic. 
Haha ！ 
And my idea about embeddings is to compute each embeddings‘ cosine for creating graphs （gnn） on prediction.
